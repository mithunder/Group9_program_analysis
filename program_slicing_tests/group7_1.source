module group71 :
read x;
prime:=true;
m:=2;
do m<x -> 
{  
	t:=x;
	{
		do t>m ->  t:= t-m
		od;
		{
			if  t=0 -> prime:=false 
			[]  t!=0 -> skip
			fi;
@programslicing="true"
			m:=m+1 
		}
	}
}
od;
write prime
end
