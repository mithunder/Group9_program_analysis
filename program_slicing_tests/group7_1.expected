module group71 :
read x;
m := 2;
do
   (m<x) ->
      m := (m+1)
od
end
